pkg/cmd/pipeline/start_test.go

```diff
@@ -285,1118 +420,22 @@ func Test_start_pipeline_interactive(t *testing.T) {
-			tb.PipelineRunStatus(
-				tb.PipelineRunStatusCondition(apis.Condition{
-					Status: corev1.ConditionTrue,
-					Reason: resources.ReasonRunning,
-				}),
-			),
-		),
-		tb.PipelineRun("pr-3", "namespace",
-			tb.PipelineRunLabel("tekton.dev/pipeline", "test"),
-			cb.PipelineRunCreationTimestamp(pr3Started),
-			tb.PipelineRunStatus(
-				tb.PipelineRunStatusCondition(apis.Condition{
-					Status: corev1.ConditionFalse,
-					Reason: resources.ReasonFailed,
-				}),
-			),
-		),
-		tb.PipelineRun("pr-1", "namespace",
-			tb.PipelineRunLabel("tekton.dev/pipeline", "test"),
-			cb.PipelineRunCreationTimestamp(pr1Started),
-			tb.PipelineRunStatus(
-				tb.PipelineRunStatusCondition(apis.Condition{
-					Status: corev1.ConditionTrue,
-					Reason: resources.ReasonSucceeded,
-				}),
-			),
-		),
-	}
-
-	ns := []*corev1.Namespace{
-		{
-			ObjectMeta: metav1.ObjectMeta{
-				Name: "namespace",
-			},
-		},
-	}
-
-	type args struct {
-		p        cli.Params
-		pipeline string
-	}
-	tests := []struct {
-		name    string
-		args    args
-		want    string
-		wantErr bool
-	}{
-		{
-			name: "lastPipelineRun Test No Err",
-			args: args{
-				pipeline: "test",
-				p: func() *test.Params {
-					clock.Advance(time.Duration(60) * time.Minute)
-
-					cs, _ := test.SeedTestData(t, pipelinetest.Data{PipelineRuns: prs, Namespaces: ns})
-					p := &test.Params{Tekton: cs.Pipeline, Clock: clock}
-					p.SetNamespace("namespace")
-					return p
-
-				}(),
-			},
-			want:    "pr-1",
-			wantErr: false,
-		},
-		{
-			name: "lastPipelineRun Test Err",
-			args: args{
-				pipeline: "test",
-				p: func() *test.Params {
-					cs, _ := test.SeedTestData(t, pipelinetest.Data{Namespaces: ns})
-					p := &test.Params{Tekton: cs.Pipeline}
-					p.SetNamespace("namespace")
-					return p
-
-				}(),
-			},
-
-			wantErr: true,
-		},
-	}
-	for _, tt := range tests {
-		t.Run(tt.name, func(t *testing.T) {
-			cs, _ := tt.args.p.Clients()
-			got, err := pipeline.LastRun(cs.Tekton, tt.args.pipeline, tt.args.p.Namespace())
-			if (err != nil) != tt.wantErr {
-				t.Errorf("lastPipelineRun() error = %v, wantErr %v", err, tt.wantErr)
-				return
-			} else if err == nil {
-				test.AssertOutput(t, tt.want, got.Name)
-			}
-		})
-	}
-}
-
-func Test_start_pipeline_newGitRes_noExistingRes(t *testing.T) {
-
-	pipelineName := "gitpipeline"
-
-	cs, _ := test.SeedTestData(t, pipelinetest.Data{
-		Pipelines: []*v1alpha1.Pipeline{
-			tb.Pipeline(pipelineName, "ns",
-				tb.PipelineSpec(
-					tb.PipelineDeclaredResource("git-repo", "git"),
-					tb.PipelineTask("first-create-file", "create-file",
-						tb.PipelineTaskInputResource("workspace", "git-repo"),
-						tb.PipelineTaskOutputResource("workspace", "source-repo"),
-					),
-					tb.PipelineTask("then-check", "check-stuff-file-exists",
-						tb.PipelineTaskInputResource("workspace", "git-repo", tb.From("first-create-file")),
-					),
-				),
-			),
-		},
+	// Without pipeline resource
```

Comment:
Could you please clarify what this comment means? There is a pipeline resource that is part of `cs2`.